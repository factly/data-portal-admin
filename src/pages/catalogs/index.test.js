import React from 'react';
import { shallow } from 'enzyme';

import CatalogList from './index';

describe('Catalogs List component', () => {
  it('should render the component', () => {
    const component = shallow(<CatalogList />);
    expect(component).toMatchSnapshot();
  });
  it('should render the component', () => {
    const component = shallow(<CatalogList />);

    const button = component.find('Button').at(0);
    expect(button.children().text()).toEqual('Add Catalog');
    const link = component.find('Link').at(0);
    expect(link.props().to).toEqual('/catalogs/create');
  });
});
