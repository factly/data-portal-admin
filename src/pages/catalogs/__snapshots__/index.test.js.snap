// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalogs List component should render the component 1`] = `
<Fragment>
  <Link
    to="/catalogs/create"
  >
    <Button
      block={false}
      ghost={false}
      htmlType="button"
      loading={false}
      style={
        Object {
          "marginBottom": 16,
        }
      }
      type="primary"
    >
      Add Catalog
    </Button>
  </Link>
  <CatalogsList />
</Fragment>
`;
